src/pages/questionPage.js

```diff
@@ -119,6 +129,22 @@ const checkAnswer = (selectedAnswer, correctAnswer, answers) => {
   })
 };
 
-const displayScore = () => {
-  document.getElementById('score').innerText = `Score: ${quizData.currentScore} out of ${quizData.questions.length}`
-}
\ No newline at end of file
+// adding a function to calculate the score
```

Comment:
displayScore function deleted,  it should be there and remove from line 132 -150